src/Yasumi/data/translations/substituteHoliday.php

```diff
@@ -12,11 +12,11 @@
 
 // Translation pattern for substitute holidays.
 return [
-    'da_DK' => '{0} (erstatning)',
-    'en_GB' => '{0} (substitute day)',
+    'da' => '{0} (erstatning)',
+    'en' => '{0} (substitute day)',
     'en_IE' => '{0} observed',
```

Comment:
Shouldn't these (en_IE, en_CA and en_US) be changed to the generic fallback locale as well?

Reply:
Well, this change was actually intentional, but I may be wrong about the preferred translation for these days in various countries.

In UK, the term “Boxing Day (substitute day)” seems the preferred variant:
https://www.gov.uk/bank-holidays, https://www.thesun.co.uk/news/1736340/august-bank-holiday-uk-2019-public/

In the US, I believe the term “observed” is used. I guess this applies to Canada as well.

I don't know about Ireland. It was previously my impression that “observed” also was used there, but now I cannot find a good reference. So I have now reverted the override for en_IE, so it falls back to the generic “XXX (substitute day)”.

Reply:
Understood. Makes sense indeed. Anyway, in case needed we can always change :)